examples/tests/sdk_tests/api/path_vars.test.w

```diff
@@ -20,14 +20,14 @@ let handler_two = inflight (req: cloud.ApiRequest): cloud.ApiResponse => {
   };
 };
 
-api.get("/users/{name}", handler);
-api.get("/path/{name}", handler);
-api.get("/users/permission/{name}", handler);
-api.get("/path/{name}/{age}", handler_two);
+api.get("/users/\{name}", handler);
+api.get("/path/\{name}", handler);
+api.get("/users/permission/\{name}", handler);
+api.get("/path/\{name}/\{age}", handler_two);
```

Comment:
Note we now need to escape these. WDYT?

Reply:
Hypothetically we could address this by choosing a different syntax for the API resource: 
```js
api.get("/users/:name", handler);
api.get("/path/:name", handler);
api.get("/users/permission/:name", handler);
api.get("/path/:name/:age", handler_two);
```

Reply:
I think we're aiming to be OpenAPI compatible so any change to API path syntax will break that.
Another thing we can do is have a raw string syntax for these cases:
`r"I'm a raw string with no escaping so this \ and { can be used freely inside me"`
`r"/path/{var}"`

Reply:
I think needing to escape these is a dealbreaker. I'm cool with `r""` although typing an extra letter doesn't feel so nice.

Reply:
Fuck

Reply:
After second thought, I think we should consider @Chriscbr's idea to use the ":foo" syntax.

We want to be OpenAPI compatible, but that doesn't imply our route syntax must be the same. We can convert it to openapi downstream. No?

Reply:
`:` is a valid character in URLs so I don't think we can get away with that one. Sadly, most useful characters are either valid URLs or otherwise reserved (like `%`, `#`, `[`, `]`)

Reply:
> `:` is a valid character in URLs so I don't think we can get away with that one.

I'm guessing this shouldn't be an issue since this is the syntax used in ExpressJS, right? If you want to represent the character `:` verbatim, I'm guessing you can escape it with `::` or something similar.

----

If the `:variable_name` syntax doesn't look Wingy, another alternative is to use angle brackets (it looks like this the syntax used by Python's [Flask](https://flask.palletsprojects.com/en/3.0.x/quickstart/) and [Django](https://docs.djangoproject.com/en/4.2/topics/http/urls/) frameworks).

```js
api.get("/users/<name>", handler);
api.get("/path/<name>", handler);
api.get("/users/permission/<name>", handler);
api.get("/path/<name>/<age>", handler_two);
```

I don't have a preference between the syntaxes. thoughts? @MarkMcCulloh @yoav-steinberg @eladb

Reply:
I think needing to escape `{` won't be intuitive to Winglang users.

Reply:
I vote for the ":field" syntax for api routes.

Reply:
done